### ISSUE TITLE
Bug after fragment restoration

### ISSUE THREAD

**Issue by MFlisar** [recruit-lifestyle/WaveSwipeRefreshLayout#20]
I have another problem. I use the view in a fragment and show the loading circle from code sometimes. This works fine until I rotate the device. Then the circle is not showing if i manually enable the refreshing (although the `isRefreshing` function returns true). But if I shortly pull down the wave view, then the loading circle is working again and I can enable refreshing from code again without issues.

So basically, after recreating the fragment I can't use the manual refreshing from code until I at least once pulled down the wave view...

**Comment by amyu** [comment]
Thanks @MFlisar !

Please wait a minute.
I will check it!!!!!
(*ﾉω・*)

**Comment by amyu** [comment]
(\*ﾉω・\*)ﾃﾍ

**Comment by amyu** [comment]
@MFlisar 

Sorry, I have not been able to grasp the situation.
Is it the fact that you use the WaveSwipeRefreshLayout in Fragment?
In order to understand quickly, please source code or sample project! ლ(´ڡ`ლ)

**Comment by MFlisar** [comment]
I now had time to investigate that... I am able to solve that problem, but I'm not sure if this brings up problems.

If I commend the line 628-630 in `WaveSwipeRefreshLayout.java` it seems to work.

Do you remember why you ignore `setRefreshing` if `mWaveView.getCurrentCircleCenterY() == 0`?
